### ISSUE TITLE
Email Support

### ISSUE THREAD

**Issue by zeLostchild** [LibreTime/libretime#724]
Email could not be sent. Check your mail server settings and ensure it has been configured properly.

1. click forgot password
2. enter information and submit form
3. See error

ERROR There is no mail server settings in the GUI!!!!

I do not believe that there is an option to add an smtp login within the user interface. Is there a config file or something? If so please let me know. If this setting was removed, will it be replaced in a future update?

**Comment by Robbt** [comment]
Yeah so for most people this probably won't work and adding the ability to use a 3rd party smtp server or what not might be handy but this is also probably a low priority in terms of development.

I think adding something in the documentation would be helpful and I'm going to change the title to reflect that this isn't a bug so much as something that we don't have good docs on how to setup and troubleshoot.

**Comment by zeLostchild** [comment]
"Alternatively, you can specify an external SMTP server in the Preferences page on Airtime's System menu." _From SourceFabrics Manual for Airtime_
It seems like it was once an option. It might be buried in the code somewhere and might be just a matter of adding it to the UI.

**Comment by Robbt** [comment]
I have found a non UI way of setting this up in PHP for people interested see [this link](https://websistent.com/how-to-use-msmtp-with-gmail-yahoo-and-php-mail/) - basically [msmtp](https://marlam.de/msmtp/) offers drop-in support for PHPs dependence upon sendmail. It could be configured with mailgun. Perhaps I'll write up the docs for how to configure this if people are interested.

**Comment by zklosko** [comment]
I found an easy way to set up the mail() function in PHP here: https://www.guru99.com/php-mail.html. Is this something we still need to add to the docs?

**Comment by marmotte32** [comment]
> I have found a non UI way of setting this up in PHP for people interested see [this link](https://websistent.com/how-to-use-msmtp-with-gmail-yahoo-and-php-mail/) - basically [msmtp](https://marlam.de/msmtp/) offers drop-in support for PHPs dependence upon sendmail. It could be configured with mailgun. Perhaps I'll write up the docs for how to configure this if people are interested.

Hi @Robbt 
Yes I'm interested on how to make work the forgot password e-mail function, could you send it to me please?

**Comment by Robbt** [comment]
I am sorry I don't recollect how to do it aside from the comment above but I plan on diving back into Libretime at least briefly to upgrade my stations instance and I'll try follow up with this if I get a chance.
